src/ipc/StoreMap.h

```diff
@@ -248,11 +248,19 @@ class StoreMap
     const Anchor &peekAtEntry(const sfileno fileno) const;
 
     /// free the entry if possible or mark it as waiting to be freed if not
-    void freeEntry(const sfileno fileno);
+    /// \param stateChanged when provided, true if the entry state changes, false otherwise
+    void freeEntry(const sfileno fileno, bool *stateChanged = nullptr);
```

Comment:
I did similarly at first, but almost all (~15) freeEntry() calls end up with ignoring its return value. We would have to prefix them with (void), which looks ugly.

Reply:
> almost all freeEntry() calls end up with ignoring its return value. We would have to prefix them with (void)

I do not think we have to or should prefix those calls with `(void)`. That prefix is useful or needed when you want to emphasize that a usually important return value is ignored on purpose rather than by accident. In this case, the return value is usually unimportant (because it does not indicate a success or failure -- the method cannot fail) and, hence, can be silently ignored by most callers.

Reply:
Done.